src/Fixtures/index.js

```diff
@@ -1434,17 +1328,35 @@ export default {
         'https://cdn3.sportngin.com/attachments/photo/be7e-106813275/Screen_Shot_2018-07-19_at_3.53.39_PM_large.png',
       description:
         "This training session aims at practicing your mark skills. Don't be broken to easily, generate dangerous passes",
-      drills: [1, 1004, 1005, 2003],
+      drills: [5, 1004, 1005, 2003],
     },
+  ],
+  blocs: [
```

Comment:
I think we talked about programs in VDD's wireframe and in the vocabulary we discussed after our last meeting : https://github.com/UltimateApp/UltimateApp/wiki#vocabulary--data-structures

Maybe we should stick to using the same word ?

Reply:
ok +1